src/lib.rs

```diff
@@ -114,6 +114,12 @@ cfg_if! {
 
         mod switch;
         pub use switch::*;
+    } else if #[cfg(target_os = "vxworks")] {
+        mod fixed_width_ints;
+        pub use fixed_width_ints::*;
```

Comment:
These two aren't necessary - should be removed.

Reply:
Is it optional? I see they are added for every target.

Reply:
It's ok to add it. The content is deprecated, so it isn't super useful to add them to new targets, but at least with them old code that uses them will "just work" for this new target, so I guess there is value in that.